lib/sgtn-client/api/translation.rb

```diff
@@ -105,6 +118,15 @@ def self.get_server(component, locale)
         return obj
       end
 
+      def self.is_json?(value)
```

Comment:
This is too costly to determine JSON, not feasible at production env.

Reply:
why it's costly? what's your suggestion to judge if a string is JSON string?

Reply:
Suggest to use hash in place of JSON string to be more efficient.

Reply:
fixed